### ISSUE TITLE
Panasonic aircon switches off when invoking Home app from Control Center

### ISSUE THREAD

**Issue by tetanus** [nitaybz/homebridge-sensibo-ac#4]
I am running version 1.0.4 of the plugin on HOOBS 3.2.4.

When my Panasonic CS-S9PKZW is running in cool or auto mode, it turns off when I pull down Control Centre and open the Home app pop-up.

There is also another issue: I can’t turn on the aircon in dry mode from the Home app pop-up. It highlights into an ‘on’ state but the highlight immediately turns off. The aircon does not respond.

The console log:

{
  "acState": {
    "on": true,
    "fanLevel": "quiet",
    "temperatureUnit": "C",
    "targetTemperature": 26,
    "mode": "dry",
    "swing": "fixedMiddleTop"
  }
}
5/16/2020, 1:14:15 PM [SensiboAC] Finished setting state for xqpToHg4:
5/16/2020, 1:14:15 PM [SensiboAC] { on: true,
  fanLevel: 'quiet',
  temperatureUnit: 'C',
  targetTemperature: 26,
  mode: 'dry',
  swing: 'fixedMiddleTop' }
5/16/2020, 1:14:15 PM [SensiboAC] Bedroom aircon AC *New State* - 
5/16/2020, 1:14:15 PM [SensiboAC] Finished updating state in HomeKit for xqpToHg4:
5/16/2020, 1:14:15 PM [SensiboAC] { on: true,
  fanLevel: 'quiet',
  temperatureUnit: 'C',
  targetTemperature: 26,
  mode: 'dry',
  swing: 'fixedMiddleTop' }
5/16/2020, 1:14:15 PM [SensiboAC] undefined
5/16/2020, 1:14:15 PM [SensiboAC] GET Response: 
 {"status":"success","result":[{"acState":{"on":true,"fanLevel":"quiet","temperatureUnit":"C","targetTemperature":26,"mode":"dry","swing":"fixedMiddleTop"},"measurements":{"batteryVoltage":null,"temperature":26.5,"humidity":35.6,"time":{"secondsAgo":28,"time":"2020-05-16T05:13:46.177038Z"},"rssi":-80,"piezo":[null,null]},"id":"xqpToHg4"},{"acState":{"on":false,"fanLevel":"medium_low","temperatureUnit":"C","targetTemperature":20,"mode":"auto","swing":"stopped"},"measurements":{"batteryVoltage":null,"temperature":31.3,"humidity":70.8,"time":{"secondsAgo":76,"time":"2020-05-16T05:12:58.087427Z"},"rssi":-60,"piezo":[null,null]},"id":"h2aNURkM"}]}
5/16/2020, 1:14:15 PM [SensiboAC] Finished updating state in HomeKit for xqpToHg4:

**Comment by nitaybz** [comment]
Does it happens when your accessories are grouped together?

**Comment by tetanus** [comment]
No, all the modes of the aircon are ungrouped.

**Comment by nitaybz** [comment]
Please install the latest version and let me know.... + add the logs please.
also I want you to please check, when the you change to DRY mode... is it changed in Sensibo app?

**Comment by tetanus** [comment]
I am using the latest version. When I change to Dry mode, yes it syncs with the Sensibo app.

However, when I turn on Dry mode in Control Centre, the icon goes off (but the aircon responds). Only after dismissing Control Centre and invoking again does the icon show that dry mode is on.

**Comment by nitaybz** [comment]
I'll try to debug... can you tell me what was the state of the aircon when you turn on the Dry mode?

**Comment by tetanus** [comment]
Thank you for trying to figure this out. The state of the aircon was dry mode.

**Comment by nitaybz** [comment]
Hey man!
Thanks for your feedback, Please install the latest @1.0.8 and let me know please

**Comment by nitaybz** [comment]
@tetanus I did a few updates to the code recently that should completely solve what you experienced. please test the latest 1.1.6 and let me know if I can close the issue

**Comment by tetanus** [comment]
This version works so much better than 1.0.11. The aircon did turn off again when I went into Home app, but this was only once in about perhaps ten tries.

**Comment by nitaybz** [comment]
Good to hear... can we close the issue? Or you still getting it 1 out of 10?
If so, I can increase the delay of a call and eliminate this 1 to 10 false state

**Comment by tetanus** [comment]
Looks like I spoke too soon. The aircon is shutting off again whenever I invoke Control Centre or go into the Home app.

**Comment by nitaybz** [comment]
add `"debug": true` to your config and show me some logs for when it happens

**Comment by nitaybz** [comment]
Please try again the latest... 1.1.11

**Comment by nitaybz** [comment]
I think I really got it this time... Please try version 1.1.15

**Comment by nitaybz** [comment]
@tetanus have you tried it again?

**Comment by tetanus** [comment]
Hi Nitay,

I think you can go ahead and close this issue. The latest version seem to be working perfectly!
Thank you again for all the effort in making it work.
